framework/src/modules/chain/logic/state_store/account_store.js

```diff
@@ -56,12 +56,14 @@ class AccountStore {
 
 	createSnapshot() {
 		this.originalData = _.clone(this.data);
-		this.updatedKeys = _.clone(this.updatedKeys);
+		this.originalUpdatedKeys = _.clone(this.updatedKeys);
 	}
 
 	restoreSnapshot() {
 		this.data = this.originalData;
-		this.updatedKeys = {};
+		this.updatedKeys = this.originalUpdatedKeys;
+		this.originalKeys = {};
```

Comment:
There is no `originalKeys` defined in the constructor and never used anywhere.

Reply:
The purpose of these changes was to reset the data stored the `originalData` and `originalUpdatedKeys`. It got messed up after the .swo file. I am fixing it, again.